http/http-advanced/pom.xml

```diff
@@ -59,7 +59,7 @@
         </dependency>
         <dependency>
             <groupId>io.quarkus</groupId>
-            <artifactId>quarkus-micrometer-registry-prometheus</artifactId>
+            <artifactId>quarkus-smallrye-metrics</artifactId>
```

Comment:
`quarkus-micrometer` is the way forward, please use micrometer instead of mp metrics.
You need to add both quarkus-micrometer + quarkus-micrometer-registry-prometheus